.gitlab-ci.yml

```diff
@@ -1,38 +1,46 @@
 image: mcr.microsoft.com/dotnet/core/sdk:3.1
 
 # Original URL format for agent artifact:
-# http://s3.amazonaws.com/dsd6-staging/windows/agent7/buildpack/agent-binaries-7.22.1-1-x86_64.zip
+# http://s3.amazonaws.com/dsd6-staging/windows/agent7/buildpack/agent-binaries-7.25.0-1-x86_64.zip
 
 # Original URL format for windows tracer home:
 # https://github.com/DataDog/dd-trace-dotnet/releases/download/v1.21.1/windows-tracer-home.zip
 
 variables:
-  AGENT_DOWNLOAD_URL: "http://apm-dotnet-throughput-cannon.eastus.cloudapp.azure.com/Builds/Custom/NamedPipesAAS/V20201231/agent-binaries-named-pipes-7.24.0.git.2.49f4f6c-1-x86_64.zip" 
-  TRACER_DOWNLOAD_URL: "http://apm-dotnet-throughput-cannon.eastus.cloudapp.azure.com/Builds/Custom/NamedPipesAAS/V20201231/windows-tracer-home.zip"
-  TRACER_CONTENT_DIR: $CI_PROJECT_DIR/content/Tracer
-  AGENT_CONTENT_DIR: $CI_PROJECT_DIR/content/Agent
+  DOTNET_AGENT_DOWNLOAD_URL: "http://apm-dotnet-throughput-cannon.eastus.cloudapp.azure.com/Builds/Custom/NamedPipesAAS/v20210108/agent-binaries-named-pipes-7.24.0.git.2.49f4f6c-1-x86_64.zip" 
+  DOTNET_TRACER_DOWNLOAD_URL: "http://apm-dotnet-throughput-cannon.eastus.cloudapp.azure.com/Builds/Custom/NamedPipesAAS/v20210108/windows-tracer-home.zip"

```

Comment:
How do we set this back to official packages?

Reply:
Manually, copy paste the link in the comment above here once we officially release.